news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "\"Medal Hunter\" Title's Second Coming"
+date: 2023-08-28 03:00:00 +0000
+---
+
+The legendary "**Medal Hunter**" title came back once again challenging every player to find a solution for a new medal!
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/banner.jpg)
+
+[Medals](/wiki/Medals) are osu! achievements which are given for achieving certain goals, and the last batch of medals was quite exciting.
+On 2023-07-27, alongside new [Hush-Hush medals](/wiki/Medals#hush-hush), [Ephemeral](https://osu.ppy.sh/users/102335) made a [news post](https://osu.ppy.sh/home/news/2023-07-27-summer-splash) in which he announced the "**Medal Hunter**" title coming back once again.
+
+"**Medal Hunter**" is a [unique user title](/wiki/People/Users_with_unique_titles), which is given for being the first player to unlock an extremely hard to solve Hush-Hush medal. The [previous "**Medal Hunter**" title](https://osu.ppy.sh/home/news/2022-09-09-how-zetera-became-a-medal-hunter) has been given out nearly **6 years ago**, which definitely made it even more interesting.
+
+In order to obtain such a desirable title, you had to be the first player to find the solution for the **Internment** medal and unlock it. **Internment** is a Hush-Hush medal added in the last batch, being so hint-based that developers found unlocking it worth a title. And while only one player could get a shiny "**Medal Hunter**" text under their username, many still have decided to team up with others on the [Medal Hunters Discord server](https://discord.gg/8qpNTs6)!
+
+The community was trying their best to *hunt* for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. *On this journey of basking in the shade of an ailing mind*, the community had received various hints, including a video recorded by one of medal hunters at [cavoe's osu! event (COE)](/wiki/Community/cavoe's_osu!_event), featuring [peppy](https://osu.ppy.sh/users/2) himself!
+
+<div align="center">
+  <video width="95%" controls>
+    <source src="https://assets.ppy.sh/media/news/coe_video_with_peppy.mp4" type="video/mp4" preload="none">
+  </video>
+</div>
+
+Within the first few days of the medal releasing, medal hunters were very quick to piece together that the hues were hinting at RGB to some capacity, but nobody knew in what way. From there, medal hunters set out to find how to input various RGB values in game, but nothing seemed to be working. In the worst of medal hunters' struggles, half of them entered full Sherlock Holmes mode during the 2nd week of the hunt, relying more on insanity than logic and drowning the discussion threads with abominations of theories. Some were genuinely good theories that Ephemeral would take note of, and others... well, let's just say Pac-Man theory.
```

Comment:
I would suggest sectioning off these paragraphs (especially after the ppy video) with different headers since it's already a cluster and a bit hard to read right now.

Another thing I would ask is a bit of an explanaion as to what and why "RGB" as well as the connection with the word "hues".

Reply:
it could be that the video itself is very large compared to the rest of the article that it becomes the first point of focus and doesn't exactly make things easy to read.

Reply:
Yea, I'm considering to change video width to 50%, because it's REALLY huge currently. As to headers, I think it would be nice, but I'm not really sure how exactly I should do that. In terms of naming and where to put them

Reply:
I'd suggest to add this image after line 26 to explain what we meant by "Pac-Man theory", but need @Walavouchey opinion on this
![pacman_theory](https://github.com/ppy/osu-wiki/assets/43725099/9298bb8b-2b0f-40c2-9eb1-8f910390b58e)

Reply:
bit adverse to the idea of mentioning [billy mitchell](https://en.wikipedia.org/wiki/Billy_Mitchell_(gamer)) here. the only relevant part is the max score of 3,333,360 points in pac-man right?

Reply:
> bit adverse to the idea of mentioning [billy mitchell](https://en.wikipedia.org/wiki/Billy_Mitchell_(gamer)) here. the only relevant part is the max score of 3,333,360 points in pac-man right?

I honestly have no idea, as I just linked an image that was originally posted on omh, and I'm not its creator

Reply:
it's just a meme and is presumably ridiculous/irrelevant on purpose

regardless, this picture is a part of the story itself and shouldn't be subject to review I think